packages/usehooks-ts/src/useIntersectionObserver/useIntersectionObserver.demo.tsx

```diff
@@ -28,7 +28,7 @@ export default function Component() {
   return (
     <>
       {Array.from({ length: 5 }).map((_, index) => (
-        <Section key={index + 1} title={`${index + 1}`} />
+        <Section key={index + 1} title={(index + 1).toFixed()} />
```

Comment:
Best to use `.toString()` here. This would cause a difference in output. Template literals coerce their values to strings, which calls the `.toString()` method instead of `.toFixed()`

```js
let x = 123.456
x.toString() // "123.456"
x.toFixed() // "123"
```

```suggestion
        <Section key={index + 1} title={(index + 1).toString()} />
```